tests/ImportTest.php

```diff
@@ -37,4 +37,12 @@ public function testBase()
 
         $this->assertSame($remoteAttr, $localAttrs[0]);
     }
-}
\ No newline at end of file
+
+    public function testKnownNamespaceLocationImport()
+    {
+//        $this->reader->addKnownNamespaceSchemaLocation('urn:example:profile-1.1', __DIR__.'/schema/profile-1.1.xsd');
```

Comment:
i think this should be un-commented

Reply:
yeah, I wanted to check travis to see that this fails. but it does not seem to run on this PR.